docs/reference/data-frames/apis/preview-transform.asciidoc

```diff
@@ -26,6 +27,13 @@ Previews a {dataframe-transform}.
 `pivot`:: Defines the pivot function `group by` fields and the aggregation to
 reduce the data.
 
+==== Authorization
+
+If the {es} {security-features} are enabled, you must have
+`manage_data_frame_transforms` cluster privileges to use this API. You must also
+have `view_index_metadata` privileges on the source index for the
+{dataframe-transform}. For more information, see
```

Comment:
`view_index_metadata` and `read` privileges on the source index.

Reply:
Thanks, I've added that privilege!